### PR TITLE
Add getChat command [fixes #22]

### LINKED ISSUE #22: "getChat" command to get information about the target chat
https://core.telegram.org/bots/api#getchat

### PR DESCRIPTION
**Pull request by artplan1**
Adding command to fetch info about chat/channel

### CODE REVIEW

File: src/commands/getchat.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          echo "Please input your TELEGRAM_BOT_TOKEN value in the CircleCI settings for this project or as a parameter for this command"
+          exit 1
+        fi
+
+        if [-z "<< parameter.telegram-chat-id >>" ]; then
+          echo "Please input your TELEGRAM_CHAT_ID value in the CircleCI settings for this project or as a parameter for this command"
+          exit 1
+        fi
+
+        curl "https://api.telegram.org/bot<< parameters.telegram-bot-token >>/getChat?chat_id=<< parameters.telegram-chat-id >>"
```
Comment (woltsu) [inline src/commands/getchat.yml]: You need to add `$` symbol in front of `<< parameters.telegram-bot-token >>`

**Comment by woltsu** [comment]
Could you check the comment above and add this to README as well please?

File: src/commands/getchat.yml (hunk 1 above)
Comment (artplan1) [reply src/commands/getchat.yml]: hey @woltsu, why `$` is necessary here? it's not variable interpolation. it should work as is

Comment (woltsu) [reply src/commands/getchat.yml]: The variable is intended to be a custom environmental variable name and not the token itself. Its purpose is to give the user an option to select the name of their own choice for the environmental variable. So if the user decides to store the token value in a variable T_TOKEN and gives that as the parameter, then the value can be accessed by typing in $T_TOKEN

Comment (artplan1) [reply src/commands/getchat.yml]: 100% makes sense, updated 👍

**Comment by woltsu** [comment]
Thanks @artplan1!

**Pull request merged**